usr/sbin/registercloudguest

```diff
@@ -327,8 +327,7 @@ def setup_ltss_registration(registration_target, regcode):
     if product is None:
         message = 'Cannot find baseproduct registration for LTSS'
         logging.error(message)
-        print(message, file=sys.stderr)
-        sys.exit(1)
+        sys.exit(message)
```

Comment:
The Python documentation [1] is ambiguous as to where the message is written, stating only
 """...if it has another type (such as a string), the object’s value is printed and the exit status is one."""

 In the existing implementation the intention is very clear, and implements the **correct** behavior, an error message goes to stderr. The change cannot guarantee this behavior since the Python specification is not explicit about how this is handled. Proof that this works as it should is missing; given the ambiguity of the Python documentation this is a non acceptable change.

[1] https://docs.python.org/3/library/exceptions.html#SystemExit